client/components/AddTestToQueueWithConfirmation/index.jsx

```diff
@@ -149,24 +149,29 @@ function AddTestToQueueWithConfirmation({
                 content={
                     'The report could not be created because an existing ' +
                     'report was found on the reports page with the same AT, ' +
-                    'browser and test plan version. Would you like to return ' +
-                    'the existing report back to the test queue?'
+                    'browser and test plan version. Work is currently ' +
+                    'underway to remove this limitation.'
```

Comment:
This makes me wonder if we don't actually want to include this change in `main` until the rest of #791 and #792 is also ready. So we can instead treat this as a current dev "base"? Do you have any thoughts on that?

Reply:
I do think it might become necessary at some point to have a branch for trend report changes. However personally I feel like this code is still mergeable to main, and we haven't arrived at the "ok now we need a separate branch" point. 

Other than removing the "proceed" button, I don't think there's much impact on the user experience. In the event the admins do want to move a report from the reports page back to the test queue, we can accommodate that through the API.

That's my perspective but I'm happy to defer to whatever you think is better.

Do you think removing the "work is underway" sentence would mitigate your concern?

Reply:
@howard-e do you want to merge it into a branch with a nice name like "trends"?

Reply:
Sure!